pandera/io.py

```diff
@@ -191,24 +192,37 @@ def _deserialize_schema(serialized_schema):
     # pylint: disable=import-outside-toplevel
     from pandera import Check, Column, DataFrameSchema, Index, MultiIndex
 
-    columns, index, checks = None, None, None
-    if serialized_schema["columns"] is not None:
+    # GH#475
+    serialized_schema = serialized_schema if serialized_schema else {}
+
+    # pylint: disable=isinstance-second-argument-not-valid-type
+    if not isinstance(serialized_schema, Mapping):
```

Comment:
You should use [collections.abc.Mapping](https://docs.python.org/3/library/collections.abc.html#collections.abc.Mapping) to make the pylint error disappear.

Reply:
I thought they were interchangeable, but I guess not. Fixed.